apps/remixdesktop/src/lib/slither.ts

```diff
@@ -0,0 +1,193 @@
+import { existsSync, readFileSync, readdirSync, unlinkSync } from 'fs'
```

Comment:
this diff shouldn't be part of the PR

Reply:
this comes from master. 
How to handle that? complete removal?

Reply:
is it? I don't see it in master

Reply:
@STetsing remove the file